_docs-v4/custom-views/custom-view-with-js.md

```diff
@@ -45,19 +48,20 @@ class CustomView extends View {
 
 export default createPlugin({
   views: {
-    custom: CustomView
+    NameOfMyCustomViewHere: NameOfMyCustomViewHere
   }
 });
 ```
 
-Then, in another file:
+
+Then, in another file that holds the main calendar instantiation logic, import your new custom view from NameOfMyCustomViewHere.js:
 
 ```js
-import customViewPlugin from './custom-view-file.js';
+import NameOfMyCustomViewHere from './NameOfMyCustomViewHere.js';
```

Comment:
I'd like to show people that what they are importing here is a plugin object, not the view class. `customViewPlugin` seems pretty appropriate here